docs/contributing/coding-standards/components.md

```diff
@@ -10,19 +10,19 @@ Generally, folder and file names should be singular, for example ‘accordion’
 
 When creating your component, you should create the following files in the component’s folder:
 
-      - `README.md`
-      - `_[component-name].scss`
-      - `_index.scss`
-      - `[component-name].yaml`
-      - `macro.njk`
-      - `template.njk`
-      - `template.test.js`
+- `README.md` - Light documentation pointing to installation instructions and component documentation on <https://design-system.service.gov.uk/>
+- `_[component-name].scss` - An SCSS file to generate the styles just for the component, importing the variables, functions and mixins from `../../base` and the styles of the component (`./index.scss`).
+- `_index.scss` - The actual styles for the component, that can be imported either on their own by `[component-name].scss` or alongside other components in `components/_all.scss` (or by the consuming application)
+- `[component-name].yaml` - Lists the options expected by the component, used for display in the documentation. It also contains the option samples used to generate the examples in the documentation, as well as the [fixtures for testing alternative implementations of the design system](https://frontend.design-system.service.gov.uk/testing-your-html/).
```

Comment:
```suggestion
- `[component-name].yaml` - Lists the component's Nunjucks macro options, which are added to the component's documentation. It also contains the option samples used to generate the examples in the documentation, as well as the [fixtures for testing alternative implementations of the design system](https://frontend.design-system.service.gov.uk/testing-your-html/).
```

Reply:
While it's correct that these are the Nunjucks macro options, we could possibly be more generic here, for the theoretical case where we ditch Nunjucks in favour of something else, these options would probably still be needed for whatever else we decided to use.

Maybe something like "Lists the parameters that can be passed to the component" (or less of a mouthful!).

Reply:
OK, sounds sensible. I like your new sentence, but do you think it would be worth adding the Nunjucks macro as an example. Maybe like - "Lists the parameters (for example, Nunjucks macros) that are passed to the component." ? This sentence format also assumes that they definitely are passed - is that right or are they a possible, rather than a definite, action?

Reply:
Yes, that's the one. I'll edit the suggestion to your description.

Reply:
I've tweaked my wording - I was conflating the Design System website (which does use the options in its documentation), with this repo specifically.